xyz-psql-connector/src/main/java/com/here/xyz/psql/tools/ECPSTool.java

```diff
@@ -56,4 +64,44 @@ public static String encrypt(String phrase, String data) throws GeneralSecurityE
   public static String decrypt(String phrase, String data) throws GeneralSecurityException {
     return new AESGCMHelper(phrase).decrypt(data);
   }
+
+  public static DatabaseSettings readDBSettingsFromECPS(String ecps, String passphrase) throws CannotDecodeException{
```

Comment:
Was this moved here from the PSQL connector config code? If not, could we change also the PSQL connector to use this directly from here?